packages/extension-sample/src/setup-server.ts

```diff
@@ -8,11 +8,15 @@
 import { IExpressApplicationExtension } from '@salesforce/pwa-kit-runtime/ssr/server/extensibility/types'
 import { Application } from 'express'
 
+// TODO: type the extensionConfig
+// NOTE: Ben will update this to use an abstract class
 class SampleExtension implements IExpressApplicationExtension {
     private options: any
+    private extensionConfig: any
```

Comment:
Are we able to type this extensionConfig property?

Reply:
We can now. Now that your PR for the abstraction class has been merged.